urllib3/response.py

```diff
@@ -269,7 +270,18 @@ def read(self, amt=None, decode_content=None, cache_content=False):
             except HTTPException as e:
                 # This includes IncompleteRead.
                 raise ProtocolError('Connection broken: %r' % e, e)
+        except Exception:
```

Comment:
I think this can be

```diff
-        except Exception:
+        except:
```

@Lukasa Is that correct/idiomatic these days? (Too much Go in my brain.)